Tasks/JavaToolInstaller/task.json

```diff
@@ -0,0 +1,153 @@
+{
+    "id": "C0E0B74F-0931-47C7-AC27-7C5A19456A36",
+    "name": "JavaToolInstaller",
+    "friendlyName": "Java Tool Installer",
+    "description": "Acquires a specific version of Java from a user supplied Azure blob or the tools cache and adds it to the PATH. Use this task to change the version of Java used in Java tasks.",
+    "helpMarkDown": "[More Information]()",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 126,
+        "Patch": 0
+    },
+    "preview": true,
+    "satisfies": ["Java"],
+    "demands": [],
+    "groups": [
+        {
+            "name":"JavaInAzureGroup",
+            "displayName": "Download Java from an Azure blob",
+            "isExpanded": true
+        }
+    ],
+    "instanceNameFormat": "Use Java $(versionSpec)",
+    "inputs": [
+        {
+            "name": "versionSpec",
+            "type": "string",
+            "label": "JDK Version",
+            "defaultValue": "1.8",
+            "required": true,
+            "helpMarkDown": "A number that specifies the JDK version to make available on the path."
+        },
+        {
+            "name": "jdkArch",
+            "type": "pickList",
+            "label": "JDK Arch type",
+            "options": {
+                "x64": "x64",
+                "x86": "x86"
+            },
+            "required": true,
+            "helpMarkDown": "The bit version of the JDK."
+        },
+        {
+            "name": "jdkSource",
```

Comment:
@lkillgore - You should closely check all the input names to make sure they meet our new standards. We shouldn't have to add aliases if we get the names right.

Reply:
👍